app/src/main/java/org/kiwix/kiwixmobile/webserver/WebServerHelper.java

```diff
@@ -0,0 +1,101 @@
+package org.kiwix.kiwixmobile.webserver;
+
+import android.content.Context;
+import android.util.Log;
+import android.widget.EditText;
+import android.widget.TextView;
+import androidx.coordinatorlayout.widget.CoordinatorLayout;
+import java.net.InetAddress;
+import java.net.NetworkInterface;
+import java.net.SocketException;
+import java.util.Enumeration;
+
+/**
+ * WebServerHelper class is used to set up the suitable environment i.e. getting the
+ * ip address and port no. before starting the WebServer
+ * Created by Adeel Zafar on 18/07/2019.
+ */
+
+public class WebServerHelper {
+  Context context;
+  private TextView textViewIpAccess;
+  private EditText editTextPort;
+  public static boolean isStarted;
+  int port;
+  private CoordinatorLayout coordinatorLayout;
+  ServerStateListener listener;
+  String TAG = WebServerHelper.this.getClass().getSimpleName();
```

Comment:
should be `private static final`, also consider just having a string here, if we ever enable proguard on debug this will not give us a meaningful tag

Reply:
Fixed. 3bb7b40be43bd943359335785dbb6d57d709db75